proofs/proof_global.ml

```diff
@@ -247,10 +169,10 @@ let start_proof sigma id ?pl str goals terminator =
     endline_tactic = None;
     section_vars = None;
     strength = str;
-    mode = find_proof_mode "No";
     universe_binders = pl } in
   push initial_state pstates
 
+(* This  *)
 let start_dependent_proof id ?pl str goals terminator =
```

Comment:
Indeed @ppedrot , you may want to have a look to the only user of this function and in particular at `Proof.dependent_start`, etc..., we are not sure this and their associated machinery should be kept in place.